cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/AbstractTicket.java

```diff
@@ -129,12 +131,12 @@ public final TicketGrantingTicket getGrantingTicket() {
     }
 
     @Override
-    public final long getLastTimeUsed() {
+    public final ZonedDateTime getLastTimeUsed() {
         return this.lastTimeUsed;
     }
 
     @Override
-    public final long getPreviousTimeUsed() {
+    public final ZonedDateTime getPreviousTimeUsed() {
```

Comment:
Don't think this will work with JPA ticket registries. We'll likely run into either schema or serialization issues. Let's keep this as long.